packages/integrations/image/src/utils/workerPool.ts

```diff
@@ -112,7 +112,7 @@ export default class WorkerPool<I, O> {
 		parentPort!.on('message', async (data) => {
 			const { msg, id } = data;
 			try {
-				const result = await cb(msg);
+				const result = cb(msg);
```

Comment:
I think we need an `await` here. It's usage at

https://github.com/joshuakgoldberg/astro/blob/e321967a408764407e0e4edd659d955a5f5db054/packages/integrations/image/src/vendor/squoosh/image-pool.ts#L148

returns a promise.

Reply:
I'd initially had https://typescript-eslint.io/rules/require-await on the enabled list & fixed up its complaints, then realized you use `await` for places that _might_ give a Promise from middleware (as stated elsewhere in the review). I missed reverting this change. Will revert this and anything else similar I can spot.